### PR TITLE
Add a method to update a repo metainformation

### PR DESCRIPTION
**Pull request by LeonidVas**
Now To update the metainformation of the repository, we can send a post request to the address that identifies the resource.
For example:
```
curl -u test:password --request POST 127.0.0.1:5000/release/2.8/ubuntu/focal
```

Supplement:
In fact, in the case of rpm based repositories will be updated repositories for all architextures including SRPM, for deb based all repositories related to this distribution.

### CODE REVIEW

File: s3repo/repoinfo.py [hunk 1]
```diff
@@ -15,3 +15,52 @@ def __hash__(self):
 
     def __eq__(self, other):
         return self.path == self.path
+
+
+class RepoAnatation:
+    """Anatation of the repository.
```
Comment (Totktonada) [inline s3repo/repoinfo.py:21]: Annotation?

File: s3repo/repoinfo.py [hunk 2]
```diff
@@ -15,3 +15,52 @@ def __hash__(self):
...
+
+    This is a more common repository description than "RepoInfo".
+    What does it mean:
+    When I want to add something (or just update the metainformation)
+    of repository "release/1.10/el/7", in fact the three repositories
+    will be updated (x86_64, SRPMS, aarch64), but it's not interesting for
+    me. Or, when I update something in the repository "release/1.10/debian/jessie"
+    the metainformation for all debian distributions will be updated.
+    So, the "RepoAnatation" describes the repository from the user's point of
+    view. And in fact, it can include more than one repository.
```
Comment (Totktonada) [inline s3repo/repoinfo.py:31]: Good description! However I miss one point regarding Debian repositories. It is clear for me what is going on for EL: x86_64, aarch64, SRPMS will be updated. But it is not clear for me what is the list for the Debian repositories. Can you, please, leave an example just like you did for EL?

Comment (LeonidVas) [reply s3repo/repoinfo.py]: Updated

File: s3repo/repoinfo.py (hunk 1 above)
Comment (LeonidVas) [reply s3repo/repoinfo.py]: yep

**Comment by Totktonada** [comment]
It is a kind of 'soft regeneration' as opposite of deleting all metainformation and generating it again. Correct? Whether we have plans to implement the latter?

**Review by Totktonada — APPROVED** [review]
No objections from me.

**Pull request merged**